src/daemon/control.rs

```diff
@@ -580,6 +586,7 @@ pub fn check_unvault_signatures(
 /// If `db_vaults` does not contain an entry for each input.
 pub fn check_spend_signatures(
     secp: &secp256k1::Secp256k1<secp256k1::VerifyOnly>,
+    managers_threshold: usize,
```

Comment:
The doc comment of this function should be updated. Also, to avoid duplication of signature checking code (and avoid using an externally provided input when possible) we should in the future just rebuild a SpendTransaction from this one by using `add_signature`s. Could you add a `TODO` here?

Reply:
Ok, the doc comment is wrong i would not normally merge this but i don't want to block it further either. Opened https://github.com/revault/revaultd/issues/253